src/frontend/magnify/packages/components/src/factories/rooms/index.ts

```diff
@@ -1,14 +1,15 @@
 import { faker } from '@faker-js/faker';
 import { Room, RoomAccessRole, RoomUserAccesses } from '../../types';
 
-export default function createRandomRoom(isAdmin: boolean = true): Room {
+export function createRandomRoom(isAdmin: boolean = true): Room {
```

Comment:
Could be good to move this modification and implied updates in a dedicated commit to remove some noise.